sim/compare_snps.py

```diff
@@ -18,25 +19,46 @@ def masked_positions(mask_filepath):
     #    why is that?
     masked_pos = []
     for i, row in mask.iterrows():
-        masked_pos.extend(list(range(row['START'], row['END']+1)))
+        masked_pos.extend(list(range(row['START'], row['END']+1)))# This is based off indexing in richards rpt.regions file - inconsisten with bedtools
 
     return masked_pos
 
-def analyse(simulated_snps, pipeline_snps, mask_filepath):
+def analyse(results_path, sample_name, mask_filepath):
     """ Compare simulated SNPs data from simuG against btb-seq's snpTable.tab
         If adjust == True: applies the mask to simulated SNPs and pipeline SNPs
         Returns a dictionary of performance stats
     """
 
+    pipeline_directory = glob.glob(results_path + 'btb-seq-results/Results_simulated-reads_*')[0] + '/'
```

Comment:
Add a TODO to DRY this up with what's in `validator.py` we should probably load data into memory in validator.py, or pass the filepath in directly to this function (leaning towards the former)